sdk/js/package.json

```diff
@@ -13,7 +13,7 @@
     "build-contracts": "npm run build --prefix ../../ethereum && node scripts/copyContracts.js && typechain --target=ethers-v5 --out-dir=src/ethers-contracts contracts/*.json",
     "build-abis": "typechain --target=ethers-v5 --out-dir=src/ethers-contracts/abi src/abi/Wormhole.abi.json",
     "build-deps": "npm run build-abis && npm run build-contracts",
-    "build-lib": "tsc -p tsconfig.json && tsc -p tsconfig-cjs.json && node scripts/copyEthersTypes.js && node scripts/copyWasm.js",
+    "build-lib": "tsc -p tsconfig.json && tsc -p tsconfig-cjs.json && node scripts/copyEthersTypes.js && node scripts/copyWasm.js && node scripts/copyAlphArtifacts.js",
```

Comment:
We could use a symbolic link: `ln -s`?

Reply:
We need to add artifacts to the release package, I'm not too familiar with how typescript release works, so I followed the same approach as the other chains.

Reply:
Sounds great then!